SDCTA/src/components/Landing_Page/body.css

```diff
@@ -35,10 +35,26 @@ body {
   border: 1px solid #7f1922;
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
   border-radius: 4px;
+  width: 100%;
 }
 
 .img img {
   display: block;
+  width: 100%;
+}
+
+.imgbox {
+  display: grid;
+  height: 100%;
+  width: 100vw;
+  margin: 0;
+  padding: 0;
+}
+.center-fit {
+  width: 100vw;
+  max-height: 100%;
+  margin: auto;
+  object-fit: fill;
```

Comment:
This is probably what was causing the images to expand weirdly when you zoom out. But when I test on an actual larger screen (27 inch), it looks good so I think this is fine.